_topics/en/coinjoin.md

```diff
@@ -84,10 +91,33 @@ see_also:
     link: topic payjoin
   - title: "Coinjoin (Bitcoin Wiki: Privacy)"
     link: https://en.bitcoin.it/Privacy#CoinJoin
+  - title: "TumbleBit: An Untrusted Bitcoin-Compatible Anonymous Payment Hub"
+    link: https://eprint.iacr.org/2016/575.pdf
+  - title: "ZeroLink: The Bitcoin Fungibility Framework"
+    link: https://github.com/nopara73/ZeroLink
+  - title: "WabiSabi: Centrally Coordinated Coinjoins with Variable Amounts"
+    link: https://github.com/zkSNACKs/WabiSabi
+  
 ---
-Named after a 2013 [proposal][gmaxwell coinjoin] by Gregory Maxwell,
-several independent implementations have provided support for various
-forms of coinjoin.
+
+On July 2nd 2011, the Bitcointalk user Hashcoin first described the concept of a collaborative transaction to achieve privacy. Later in 2013, the concept was named a coinjoin after a [proposal][gmaxwell coinjoin] by Gregory Maxwell. He laid out the ideas and set up a developer bounty to make them a reality.
```

Comment:
Is there a good link for the hashcoin post?

Reply:
Added